forest/src/cli/mod.rs

```diff
@@ -4,7 +4,7 @@
 mod auth_cmd;
 mod chain_cmd;
 mod client;
-mod config;
+pub mod config;
```

Comment:
I think it would be cleaner to re-export `LogConfig` and `LogValue` in the same way that `Config` is re-exported.

Reply:
Re-exported